src/constants/visualizationTypes.ts

```diff
@@ -47,11 +47,11 @@ export type ChartType =
     | "pareto"
     | "alluvial";
 
-export type GeoPushpinType = "pushpin";
+export type GeoType = "pushpin";
```

Comment:
We can revert this in the future if `Geo Polygon` is required to impl :)
Can you explain why don't we continue with `GeoPushpinType `?

Reply:
In the future, we will have 
`export type GeoType = "pushpin" | "polygon";`.
I'm following the previous type 
```
export type ChartType =
    | "bar"
    | "column"
...
```